release-calendar/src/client/api-service.ts

```diff
@@ -43,6 +47,13 @@ export class ApiService {
     ];
   }
 
+  async isRelease(requestedRelease: string): Promise<boolean> {
```

Comment:
Why not reuse `getRelease` and see if it turns up empty?

Reply:
I had thought that counting is better for perfomance. Is that not correct?

Reply:
It's arguable, because you're adding `getCount()` then seeing if `count > 0`.